identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```
@@ -192,6 +192,7 @@ under.construction=This page is under construction
 by.selecting.following.attributes=By selecting the following attributes you agree to share them with
 select.all=Select All
 requested.scopes=Obtain permission for
+requested.authorization-details=Gain consent for
```

Let's follow the convention for i18n keys.

Also update the other language properties files too.

```suggestion
requested.authorization.details=Gain consent for
```

---

Fixed this with a54ea18350058523e7ef006bbd3cd57fdb570968